### PR TITLE
[TRIVIAL] [PDF] Fix all occurences of $(D ...) straddling the end of a line

### PR DESCRIPTION
**Pull request by andralex**

### CODE REVIEW

File: spec/class.dd [hunk 1]
```diff
@@ -371,8 +371,8 @@ $(GNAME Constructor):
         ------
 
         $(P If no call to constructors via $(D this) or $(D super) appear in a
-        constructor, and the base class has a constructor, a call to $(D
-        super)() is inserted at the beginning of the constructor.)
+        constructor, and the base class has a constructor, a call to
+        `super()` is inserted at the beginning of the constructor.)
```
Comment (wilzbach) [inline spec/class.dd]: Before:

image


After:

image

File: spec/hash-map.dd [hunk 2]
```diff
@@ -97,8 +97,8 @@ $(H2 $(LNAME2 using_classes_as_key, Using Classes as the KeyType))
         that are considered equal should always have the same hash value. If
         this is not the case, the associative array will not function properly.
         Also note that $(D opCmp) is not used to check for equality by the
-        associative array. However, since the actual $(D opEquals) or $(D
-        opCmp) called is not decided until runtime, the compiler cannot always
+        associative array. However, since the actual $(D opEquals) or
+        `opCmp` called is not decided until runtime, the compiler cannot always
```
Comment (wilzbach) [inline spec/hash-map.dd]: Before:

image

After:

image

File: spec/operatoroverloading.dd [hunk 3]
```diff
@@ -594,9 +594,9 @@ void test()
 $(H3 $(LNAME2 slice_assignment_operator, Slice Assignment Operator Overloading))
 
 	$(P If the left hand side of an assignment is a slice operation on a
-	struct or class instance, it can be overloaded by implementing an $(D
-	opIndexAssign) member function that takes the return value of the $(D
-	opSlice) function as parameter(s).
+	struct or class instance, it can be overloaded by implementing an
+	`opIndexAssign` member function that takes the return value of the
+	`opSlice` function as parameter(s).
```
Comment (wilzbach) [inline spec/operatoroverloading.dd]: Before:

image

After:

image

**Comment by andralex** [comment]
wish we had an LTRIM macro that trimmed the leading whitespace off of its argument

**Pull request merged**